plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/thrift/ThriftHiveMetastore.java

```diff
@@ -1328,16 +1332,20 @@ private void alterPartitionWithoutStatistics(HiveIdentity identity, String datab
         }
     }
 
+    // TODO: Today, this method is called exclusively for addPartition or alterPartition.  For transactional tables,
+    // the column statistics will be empty.  Issue #7190 calls for creating stats for transactional INSERT, DELETE and UPDATE.
```

Comment:
Use `TODO (https://github.com/trinodb/trino/issues/7190)` when linking to an issue

Reply:
> Today, this method is called exclusively for addPartition or alterPartition

Thanks for using "Today", as it can server as a reminder to the reader, that the information can become outdated.
Still, some readers may overlook this suggestion. I wouldn't assume this comment will get updated when a new call site is introduced, although i obviously should.